.github/workflows/run_slow_unit_tests.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
     steps:
     - uses: actions/checkout@v2
```

Comment:
I think we'll want to update these, as well. (i.e. `actions/checkout` -> `@v4`)

Reply:
Thanks for catching this! Apparently need more coffee this morning 😅